tests/QuikGraph.Tests/Algorithms/Contracts/VertexColorizerContract.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using QuikGraph.Algorithms;
+using QuikGraph.Algorithms.MaximumFlow;
+using QuikGraph.Algorithms.ShortestPath;
+using QuikGraph.Tests.Algorithms.MaximumFlow;
+using QuikGraph.Tests.Algorithms.ShortestPath;
+
+// ReSharper disable ReturnValueOfPureMethodIsNotUsed
+
+namespace QuikGraph.Tests.Algorithms.Contracts
+{
+    public static class Feeder
+    {
+        public static IEnumerable<Type> VertexColorizers
```

Comment:
This most likely should be extracted to a helper method that can deal with both generic and non-generic interfaces. Then we can add other interfaces here also and move the Feeder class out